src/shared_utilities/Domains.jl

```diff
@@ -721,6 +746,27 @@ function get_Δz(z::ClimaCore.Fields.Field)
     return Δz_top ./ 2, Δz_bottom ./ 2
 end
 
+"""
+    get_z_sfc(z::ClimaCore.Fields.Field)
+
+A function to return the coordinates of the surface `z`, projected onto the one-/two-dimensional surface space.
+This space is distinct from the level space of `z`.
```

Comment:
It would be good to note that this function is non-allocating, since it constructs a Field of field_values and a space that both already exist

Reply:
I think it allocates z_face, but other than that is non-allocating. I left that comment!